### ISSUE TITLE
Feature request: parse sass functions for colors

### ISSUE THREAD

**Issue by talamaska** [KamiKillertO/vscode-colorize#75]
Hi, I have activated the experimental feature 'activate_variables_support_beta'
So far it works as expected. I use quite a lot the sass function for generating new colors like darken, lighten, saturate, desaturate , adjust-hue, tint, shade and of course rgba in form rgba($base-color, 0.7).
Would it be possible to actually display the generated colors? The integrated color previewer in VS code doesn't understand the variable value and show black. obviously there is a syntax difference from sass rgba and css rgba, would it be possible to change the behavior according to the syntax?

**Comment by KamiKillertO** [comment]
I've planned to support sass, less, stylus... function. But it will be later for now I'm focused on variables support

**Comment by talamaska** [comment]
thanks

**Comment by KamiKillertO** [comment]
Yes

**Comment by cj81499** [comment]
Okay. Any idea when this might be implemented? I'd really love it as a feature.

**Comment by KamiKillertO** [comment]
Not really, I'm still working on variables support I've still some things to do 😞.
But if you have any idea about how integrate it you can still make a PR 😉.

**Comment by Arkkimaagi** [comment]
How about starting with support for rgba?
```
![image](https://user-images.githubusercontent.com/1191667/55055459-a61aae80-506b-11e9-84e1-beee93126b10.png)
```

**Comment by ninthart** [comment]
Is there any news on this? It's the one thing I really miss from Atom's Pigments extension…

**Comment by racha** [comment]
Literally only thing in the way of me switching to VS Code.... Please add support for those so it looks like this

![image](https://user-images.githubusercontent.com/3868640/76168907-ef2c1b80-6173-11ea-8354-10b159690889.png)


Also minimap colors would be amazing

**Comment by mrleblanc101** [comment]
Any update on this ?
I was using enyancc/vscode-ext-color-highlight before, but it doesn't support variable.
Now I switched to KamiKillertO/vscode-colorize but it doesn't support functions.
Seem like both worked fine in Atom.
